### ISSUE TITLE
Slow continous queries

### ISSUE THREAD

**Issue by bovy89** [influxdata/influxdb#6788]
Hi, I'm facing an issue with continuous queries, because they take too much time to complete their execution.

I have the following continuous queries:

```
CREATE CONTINUOUS QUERY populate_one_month ON mydb BEGIN SELECT mean("value") AS "value", mean("duration") AS "duration" INTO "one_month".:MEASUREMENT FROM /./ GROUP BY time(15m),domain,host,metric fill(none) END
CREATE CONTINUOUS QUERY populate_six_months ON mydb BEGIN SELECT mean("value") AS "value", mean("duration") AS "duration" INTO "six_months".:MEASUREMENT FROM /./ GROUP BY time(30m),domain,host,metric fill(none) END
CREATE CONTINUOUS QUERY populate_thirteen_months ON mydb BEGIN SELECT mean("value") AS "value", mean("duration") AS "duration" INTO "thirteen_months".:MEASUREMENT FROM /./ GROUP BY time(1h),domain,host,metric fill(none) END
```
I've tried to execute one of them manually:
```
SELECT mean("value") AS "value", mean("duration") AS "duration" INTO "thirteen_months".:MEASUREMENT FROM /./ WHERE time >= '2016-06-03T08:30:00Z' AND time < '2016-06-03T09:30:00Z' GROUP BY time(1h),domain,host,metric fill(none)
```

Without the into clause, it works fine and return in a few of seconds;
With the into clause, it tooks more than 25m to complete and then influx cli says "177104 written".

InfluxDB Version: 0.13
OS: CentOS release 6.7

I'm not observing any cpu or disk workload.

I don't know if it is related to this issue, https://github.com/influxdata/influxdb/issues/6685, but I've tried the nightly release and it didn't resolve my issue.

**Comment by toddboom** [comment]
@bovy89 Would it be possible for you to provide your dataset for us to test against?

**Comment by jsternberg** [comment]
This is likely fixed by #7007. This should be in 1.0.0-beta3. If you still have issues with this, please comment so I can reopen this ticket. Thanks.
